__tests__/app.test.js

```diff
@@ -48,7 +48,7 @@ describe("GET /api/articles/:article_id", () => {
           body: expect.any(String),
           created_at: expect.any(String),
           votes: expect.any(Number),
-          comment_count: expect.any(String),
+          comment_count: "11",
```

Comment:
This should be a number, not a string please! Think back to the session earlier where we talked about how to change types in sql (clue - you'll need `::` )

Reply:
this is still a string, it needs to be a number not just a string of a number